docs/ml-features.md

```diff
@@ -145,9 +148,11 @@ for more details on the API.
  passed to other algorithms like LDA.
 
  During the fitting process, `CountVectorizer` will select the top `vocabSize` words ordered by
- term frequency across the corpus. An optional parameter "minDF" also affects the fitting process
+ term frequency across the corpus. An optional parameter `minDF` also affects the fitting process
  by specifying the minimum number (or fraction if < 1.0) of documents a term must appear in to be
- included in the vocabulary.
+ included in the vocabulary. Another optional binary toggle parameter controls the output vector.
```

Comment:
Hmm, it's strange as I still get different results (clean build off latest master @ `1360a6d636dd812a27955fc85df8e0255db60dfa`):

<img width="871" alt="screen shot 2016-05-30 at 8 45 01 am" src="https://cloud.githubusercontent.com/assets/1036807/15643010/d318b6bc-2642-11e6-9ca8-d2825bd1dcce.png">

For now we can leave the example as is, but it's a little worrying to be getting this difference. Will need to dig further - can you confirm which commit hash you're building off?